test/functionalTest/cases/1187_empty_attribute_in_create/empty_attribute_in_create.test

```diff
@@ -120,11 +120,6 @@ Date: REGEX(.*)
         "type": null,
         "value": ""
     },
-    "A2": {
```

Comment:
I understand that the "side effect" which causes A2 not being shown, should be fixed.